release_notes/v9_0_0.html.md

```diff
@@ -1,141 +1,275 @@
+* Theme based styling provided by Styled Component. Classic, Small, Medium and Large theme styles. (Components: Day Picker, Date Navbar Button, Date Weekday, Date Navbar)
+* Theme based styling provided by Styled Component. Classic, Small, Medium and Large theme styles. (Component: ActionToolbar)
+* Theme based styling provided by Styled Component. Classic, Small, Medium and Large theme styles. (Component: Pill)
+* Theme based styling provided by Styled Component. Classic, Small, Medium and Large theme styles. (Component: SimpleColorPicker)
 
 ### Other
-* add docs to Storybook on how to override the theme
-* Removes the theme object from the component snapshot tests
-* Replaces Save and Cancel Button with a generic Form Button
+* Added validation API. (Components: Form, Input, Select, Textarea, Textbox, Dialog)
+* Changed to remove reliance on Input decorator. (Components: ButtonToggle, ButtonToggleGroup)
+* Changes component to allow removal of InputDecoratorBridge and replace with Textbox. (Component: Select)
+* Added docs to Storybook on how to override the theme. (Component: Theme)
+* Changed Alert to include default props and other improvements. (Component: Alert)
+* File naming of component specs. (Component: RadioButton)
+* Changes to remove the theme object from the component snapshot tests.
+* Added further test specs. (Components: Dialog, Dropdown, DropdownFilter, DropdownFilterAjax, Flash)
+* Added a new validations story (Component: Select)
+* Added style tests for experimental Form. Replaced Save and Cancel Button with a generic Button. (Component: Form)
+* Updates `.renogen` headings to include a `Deprecated Components` option. Creates a `__deprecated__` directory.
+* Added support for 'type' prop to add color. It also now repositions itself automatically if rendered off screen. (Component: Tooltip)
+* Changed to remove reliance on Input decorator. (Component: SimpleColorPicker)
 
-### Bug fixes
-* Adds classic theme and fixes carbon demo site styling bugs (sidebar, header, alert-banner, sage loves carbon).
-
-### Improvement
-* Add styled components to Spinner. Create new loader component.
-* Add styled components to Tabs Component.
-
-### Storybook Update
-* Fixed documentation 'info' and 'note' implementation to avoid duplication that was triggering not unique key console warning.
-
-### Demo Site Updates
-* add Google Analytics to Storybook
+### New Components
+* Added a Switch component. (Component: Switch)
+* Added new Input Icon Toggle component based on the input-icon decorator. Toggle icon can be added to input components. (Component: Input Icon Toggle)
+* Added a DateInput component based on the Textbox component. (Component: DateInput)
+* Added a Message component. (Component: Message)
+* Added a Loader component. (Component: Loader)
+* Added new RadioButtonGroup component. (Component: RadioButtonGroup)
+* Added a Decimal component based on the existing implementation. (Component: Decimal)
+* Adds a Card Component
```

Comment:
For consistency with the other items in this section, maybe this should be reworded to:
```suggestion
* Added a Card component. (Component: Card)
```
Source file – `change_log/next/FE-1517-card-component.yml`

Reply:
Thanks for fixing this in 8fa2ea0 :thumbsup: